.github/workflows/wasmcloud.yml

```diff
@@ -522,7 +522,7 @@ jobs:
           docker run --rm wasmcloud.azurecr.io/${{ steps.ctx.outputs.owner }}/${{ matrix.bin }}:${{ steps.ctx.outputs.version }}-wolfi ${{ matrix.bin }} --version
 
       - name: Push `${{ matrix.bin }}` `latest` tag
-        if: startswith(github.ref, format('refs/tags/{0}v', matrix.prefix)) && !steps.ctx.outputs.prerelease
+        if: startswith(github.ref, format('refs/tags/{0}v', matrix.prefix))
```

Comment:
I think we should keep this, e.g. latest isn't overwritten by prerelease

Reply:
Done!